src/lib/accessibility/FocusHandling/FocusContainer.ts

```diff
@@ -1,92 +1,66 @@
 import * as _ from 'lodash'
-import { FocusableItemProps, SetStateDelegate } from './FocusableItem'
 
-export interface FocusContainerProps<T> {
-  items?: T[]
-}
+export class ContainerFocusHandler {
+  private _focusedItemIndex = 0
 
-export interface FocusContainerState {
-  focusItemOnIdx: number
-}
+  constructor(private getItemsCount: () => number, private readonly setFocusAt: (number) => void) {}
+
+  private coerceFocusedItemIndex() {
```

Comment:
nit: can we use a simpler vocabulary for this method name?

Reply:
please, introduce your suggestions for that

Reply:
```constrainFocusedItemIndex``` ?

Reply:
yes, this is exact the rename made 👍